source/hash_table.c

```diff
@@ -833,8 +833,16 @@ uint64_t aws_hash_ptr(const void *item) {
     return ((uint64_t)b << 32) | c;
 }
 
-bool aws_c_string_eq(const char *a, const char *b) {
-    return !strcmp(a, b);
+bool aws_hash_callback_c_str_eq(const void *a, const void *b) {
```

Comment:
You don’t need to cast to and from void*

Reply:
Fixed